source/developers/component_loading.markdown

```diff
@@ -9,17 +9,15 @@ sharing: true
 footer: true
 ---
 
-A component will be loaded on start if a section (ie. `light:`) for it exists in the config file. A component can also be loaded if another component is loaded that depends on it. When loading a component Home Assistant will check the following paths:
+A component will be loaded on start if a section (ie. `light:`) for it exists in the `configuration.yaml` file. A component can also be loaded if another component is loaded that depends on it. When loading a component Home Assistant will check the following paths:
 
- * `<config directory>/custom_components/<component name>`
+ * `.homeassistant/custom_components/<component name>`
```

Comment:
I do not agree on this change, since it is not Hass.io compatible.
The previous version works better.